PORTALS-3253 - Remove imports against `src/index.ts` (#1553)
Sage-Bionetworks/synapse-web-monorepo: packages/synapse-react-client/README.md

## Adding a new component and Publishing a new version of SRC

> JavaScript packages and applications inside the monorepo can directly import modules within the `src` folder, so modules created within the `src` directory are accessible from other TypeScript packages and applications within the monorepo--they have been configured to reference the synapse-react-client source files.

Author: ```suggestion
JavaScript packages and applications inside the monorepo that have been configured to reference the synapse-react-client source files can directly import modules within the `src` folder without exporting them from any additional file.
```
